src/modal/modal.js

```diff
@@ -130,36 +130,32 @@ angular.module('ui.bootstrap.modal', ['ui.bootstrap.multiMap', 'ui.bootstrap.sta
               }).start().then(done);
             });
           }
+          // Notify {@link $modalStack} that modal is rendered.
+          var modal = $modalStack.getTop();
+          if (modal) {
+            $modalStack.modalRendered(modal.key);
```

Comment:
This changes the business logic of `rendered`. `opened` is meant for what it appears `rendered` has been changed to in this PR. This is no good.